### PR TITLE
🚧 Admin Portal updates

### PR DESCRIPTION
**Pull request by JackDevAU**
## Should be merged with this PR: [#3976 - Admin Portal](https://dev.azure.com/ssw/SSW.Rewards/_git/SSW.Rewards.Admin/pullrequest/3976)

- Skill Upsert & Delete Routes
- Staff fix skill updates
- fixes for notifications routes
- db migrations fixes

### CODE REVIEW

File: API/SSW.Rewards.Persistence/Migrations/20220607061234_UnFuckDatabase.cs [hunk 1]
```diff
@@ -3,10 +3,51 @@
 
 namespace SSW.Rewards.Persistence.Migrations
 {
-    public partial class AddRoles : Migration
+    public partial class UnFuckDatabase : Migration
```
Comment (matt-goldman) [inline API/SSW.Rewards.Persistence/Migrations/20220607061234_UnFuckDatabase.cs:6]: Are we merging this?

Please rename this migration!

**Review by matt-goldman — CHANGES_REQUESTED** [review]
Looks good but there are a couple of changes required

File: API/SSW.Rewards.Persistence/Migrations/20220607061234_UnFuckDatabase.cs (hunk 1 above)
Comment (william-liebenberg) [reply API/SSW.Rewards.Persistence/Migrations/20220607061234_UnFuckDatabase.cs:6]: Yes, please change this :)

Comment (wicksipedia) [reply API/SSW.Rewards.Persistence/Migrations/20220607061234_UnFuckDatabase.cs]: resolved

File: API/SSW.Rewards/Controllers/BaseController.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 namespace SSW.Rewards.WebAPI.Controllers
 {
-    [Authorize]
+    //[Authorize]
```
[the remark this answers is not in the document]
Comment (wicksipedia) [reply API/SSW.Rewards/Controllers/BaseController.cs]: reverted change

**Review by matt-goldman — APPROVED** [review]

**Pull request merged**